plugins/newsdownloader.koplugin/main.lua

```diff
@@ -20,6 +20,35 @@ local FILE_EXTENSION = ".html"
 local FEED_SOURCE_SUFFIX = "_rss_tmp.xml"
 local NEWS_DL_DIR, FEED_CONFIG_PATH
 
```

Comment:
I think it would be good to have here _"/news/"_ extracted (from code below) to a variable
 _NEWS_DL_DIR = DataStorage:getDataDir() .. "/news/"_
so I could be easly found where to change the default dir (e.g. on Android), and not have to search throughout all implementation

e.g. 
_local DEFAULT_NEWS_DIR_NAME = "/news/"_
and
 _NEWS_DL_DIR = DataStorage:getDataDir() .. DEFAULT_NEWS_DIR_NAME_